schema/deploy/function_current_timestamp.sql

```diff
@@ -3,9 +3,9 @@
 
 begin;
 
-create function ggircs_portal.current_timestamp() returns timestamptz as
+create or replace function ggircs_portal.current_timestamp() returns timestamptz as
```

Comment:
Instead of updating this function, I would delete it and replace it with `now()` where it is used

Reply:
This looks like it would involve rewriting a lot of tests...is it worth it at this time?

Reply:
Could be done later. If that's the case, let's make sure we don't forget to do it by adding a tech debt card

Reply:
Here it is! I think it makes sense to split the work anyways https://youtrack.button.is/issue/GGIRCS-2054